.rubocop_todo.yml

```diff
@@ -1,28 +1,34 @@
-# This configuration was generated by `rubocop --auto-gen-config`
-# on 2015-07-14 22:11:05 -0400 using RuboCop version 0.32.1.
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-04-11 14:03:18 -0700 using RuboCop version 0.34.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 2
+# Offense count: 33
 Metrics/AbcSize:
-  Max: 120
+  Max: 100
```

Comment:
FWIW, I had included some padding in some of these because it was driving me nuts that every time someone sent a PR we had to update the rubocop config.  I also pinned rubocop to a specific version to prevent spurious failures.